### PR TITLE
Pull in upstream updates

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request updates this repository with the latest in [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library). This includes a configuration for [cisagov/action-lineage](https://github.com/cisagov/action-lineage) so it will get future updates automatically.

## 💭 Motivation and context ##

I was reminded of this project this week and noticed that it was woefully out of date.

## 🧪 Testing ##

Automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
Thanks for bringing this one back to life, necromancer!

File: bump_version.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# bump_version.sh (show|major|minor|patch|prerelease|build)
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+VERSION_FILE=src/example/_version.py
```
Comment (dav3r) [inline bump_version.sh]:
```suggestion
VERSION_FILE=src/lcgit/_version.py
```

**Review by dav3r — CHANGES_REQUESTED** [review]
Please see the one small thing that I mentioned, then we're good to go.  Thanks for shepherding this lost lamb back to the flock! 🐑

File: bump_version.sh (hunk 1 above)
Comment (mcdonnnj) [reply bump_version.sh]: Ah! Missed that one, thanks!

**Review by dav3r — APPROVED** [review]
🤘

**Pull request merged**